packages/vscode/src/index.ts

```diff
@@ -19,12 +19,17 @@ export async function activate(ext: ExtensionContext) {
   log.appendLine(`UnoCSS for VS Code  v${version} ${process.cwd()}`)
 
   const contextLoader = new ContextLoader(cwd)
+
+  const hasConfig = await contextLoader.loadContextInDirectory(cwd)
+  if (!hasConfig)
```

Comment:
The `load` function in `loadContextInDirectory` may be called twice, because `new ContextLoader` calls loadContextInDirectory by default, so it's better to call `contextLoader.loadContextInDirectory` after `contextLoader.ready`, and this call will hit the `contextsMap` cache (the cache value is `null`)